translations/de/info/task_description.html

```diff
@@ -1,37 +1,38 @@
 <p>
-    If you have solved the
+    Wenn Du die Mission
     <a href="http://www.checkio.org/mission/find-friends/share/3061fec1b35588e4bae8ad69db073aab/">
         "How to find friends"</a>
-    mission, then you already know how to check for the existence of a path
-    in graphs. Let's try to add something more to that problem.
+    gelöst hast, weißt Du schon wie Du das Vorhandensein eines Weges in einem Diagramm prüfst.
+    Wir werden hier dieses Problem noch etwas ausbauen.
 </p>
 
 <p>
-    You are given a matrix (2D array) and the coordinates (row and column) of two cells with the same value. The matrix
-    consists of digits. You may move to neighbouring cells either horizontally or vertically provided the values of the
-    origin and destination cells are equal. You should determine if a path exists between two given cells.
+    Gegeben ist eine Matrix (2D Array) und die Koordinaten (Reihe und Spalte) zweier Zellen mit identischem Wert. Die 
+    Matrix besteht aus Zellen mit Zahlenwerten. Zu benachbarten Zellen kann man horizontal oder vertikal gelangen wenn die Werte identisch sind.
+    Du sollst herausfinden ob ein Weg existiert zwischen den zwei gegebenen Zellen.
 </p>
 
 <p>
-    A matrix is represented as a tuple of tuples with digits. Coordinates are represented as a tuple with two numbers:
-    row and column. The result should be any value which can be converted into a boolean. If a path exists, then return
-    True. Return False if there is none.
+    Die Matrix besteht aus einer Tuple mit Tuplen von Zahlenwerten. Eine Tuple mit zwei Zahlen repräsentiert die Koordinaten in 
+    Reihe und Spalte. Das Ergebnis muss als ein booleschen Wert interpretierbar sein. Gib True zurück wenn ein Weg existiert, 
+    False, wenn nicht.
 </p>
 
 <p style="text-align: center">
     <img src="{{ MEDIA }}can-jump-through.svg" alt="can-you-jump-through" style="max-width: 430px">
 </p>
 
 <p>
-    <strong>Input: </strong>
-    Three arguments. A matrix as a tuple of tuples with integers,
-    first and second cell coordinates as tuples of two integers.
+    <strong>Eingabe: </strong>
+    Drei Argumente. Eine Matrix als Tuple mit Tuplen voller Integern, sowie die Koordinaten der 
+    ersten und zweiten Zelle als Tuple zweier Integer.
 </p>
 
-<p><strong>Output: </strong>The existence of a path between two given cells
-    as a boolean or a value that can be converted to boolean.</p>
+<p><strong>Ausgabe: </strong>Ein boolescher Wert oder ein Wert der in einen boolerschen Wert konvertiert werden kann,
```

Comment:
**Zu:** _Ein boolescher Wert oder ein Wert, der in einen booleschen Wert konvertiert werden kann und angibt, ob ein Weg zwischen den zwei gegebenen Zellen existiert._  
 ==>